examples/server_profiles.py

```diff
@@ -169,6 +169,22 @@
 except HPOneViewException as e:
     print(e.msg)
 
+try:
+    # Get Server Profile by scope_uris
+    if oneview_client.api_version == 600:
+        enclosure_group_uri = "/rest/enclosure-groups/8cf8fd62-ad9f-4946-abf7-6dac9cb59253"
+        server_hardware_type_uri = "/rest/server-hardware-types/B342B5D4-387D-4DEB-ADBB-9D7256DF2A47"
+        available_networks = oneview_client.server_profiles.get_available_networks(enclosureGroupUri=enclosure_group_uri,
+                                                                                   serverHardwareTypeUri=server_hardware_type_uri, view='Ethernet',
+                                                                                   scope_uris="\"'/rest/scopes/3bb0c754-fd38-45af-be8a-4d4419de06e9'\"")
+        if len(available_networks) > 0:
+            print("Found Server Profiles by scope_uris: .\n  uri = '%s'" % available_networks['uri'])
+            pprint(available_networks)
+        else:
+            print("No Server Profiles Group found.")
+except HPOneViewException as e:
+    print(e.msg)
+
 try:
     # Get the list of available servers
     print("\nList all available servers associated with a server hardware type and enclosure group")
```

Comment:
This Parameter is added to 2more endpoints, available servers and available targets. Confirm once with priyanka if examples needed to be provided for them as well.

Reply:
Examples for available servers and available targets already exists.